### ISSUE TITLE
Call to a member function reply() on string - Error

### ISSUE THREAD

**Issue by sasureshkumar** [mpociot/botman#70]
```php
$bot->say('You have to verify your identity, please use this link: ' . $link, $bot->getMessage()->getUser(), BotFrameworkDriver::class);
```

Since BotMan.php Line 237
```php
public function say($message, $channel, $driver = null)
    {
        $driver = new BotFrameworkDriver();
        if (is_null($driver)) {
            $drivers = DriverManager::getConfiguredDrivers($this->config);
        } else {
            $drivers = [$driver];
        }

        foreach ($drivers as $driver) {
            $matchMessage = new Message('', '', $channel);
            /* @var $driver DriverInterface */
            $driver->reply($message, $matchMessage, []);
        }

        return $this;
    }
```

```php
$driver->reply($message, $matchMessage, []);
```
Above $driver instance is String since we send it as String from say() method and calling reply() method is throw the error.

**Comment by mpociot** [comment]
Yup, you're right. I'll fix it in the next release.

**Comment by sasureshkumar** [comment]
@mpociot Thanks lot.

**Comment by mpociot** [comment]
Fixed in version 1.2.2
